sfepy/discrete/fem/lcbc_operators.py

```diff
@@ -528,6 +526,78 @@ def __init__(self, name, regions, dof_names, dof_map_fun,
                                          ts, functions=functions)
 
 
+class MultiNodeLCOperator(LCBCOperator):
+    r"""
+    Transformation matrix operator that define the DOFs in one (depenedent) node
+    as a linear combination of the DOFs in some other (independent) nodes.
+
```

Comment:
Transformation matrix operator that defines the DOFs in one (dependent) node
    as a linear combination of the DOFs in some other (independent) nodes.